src/rpc/server.cpp

```diff
@@ -467,14 +467,43 @@ std::string HelpExampleRpc(const std::string& methodname, const std::string& arg
         "\"method\": \"" + methodname + "\", \"params\": [" + args + "] }' -H 'content-type: text/plain;' http://127.0.0.1:8232/\n";
 }
 
-string experimentalDisabledHelpMsg(const string& rpc, const string& enableArg)
-{
-    return "\nWARNING: " + rpc + " is disabled.\n"
-        "To enable it, restart zcashd with the -experimentalfeatures and\n"
-        "-" + enableArg + " commandline options, or add these two lines\n"
-        "to the zcash.conf file:\n\n"
-        "experimentalfeatures=1\n"
-        + enableArg + "=1\n";
+string experimentalDisabledHelpMsg(const string& rpc, const vector<string>& enableArgs)
```

Comment:
While we are changing this line, make these:
```suggestion
std::string experimentalDisabledHelpMsg(const std::string& rpc, const std::vector<string>& enableArgs)
```